Solutions/Infoblox NIOS/Parser/Infoblox_dnsclient.txt

```diff
@@ -30,13 +30,13 @@ let response =
         ": query: " DnsQuery: string
         " " DnsQueryClassName: string
         " " DnsQueryTypeName: string
-        " response: " DnsResponseCodeName: string
+        " response: " DnsResponseCode: string
         " " DnsFlags: string
```

Comment:
@vakohl Let's close the parity between old and new version. If there are a few fields extra in the new version, like EventProduct or EventVendor (Normalized Schema), that might not be a concern, but no field should be missing. Example, "Log_type" which is not used anywhere, is present in other Infoblox parser branch. We should add it here too.

Reply:
Thanks Devika. Made further changes to keep schema same as old parser